### PR TITLE
doc/encoders.texi Add doc for qsv

### PR DESCRIPTION
**Pull request by wenbinc-Bin**
You can use texinfo to view ffmpeg doc.

sudo apt install texinfo
makeinfo --html --no-split -o ffmpeg-codecs.html ffmpeg-codecs.texi
open ffmpeg-codecs.html.

### CODE REVIEW

**Comment by wenbinc-Bin** [comment]
hevc_qsv seems doesn't enable look_ahead on mainline, but it has an option "look_ahead_depth". I think this doesn't work for hevc so I don't add this option to doc.

File: doc/encoders.texi [hunk 1]
```diff
@@ -3212,6 +3214,242 @@ encoder use CAVLC instead of CABAC.
...
+@end table
+
+@item @var{forced_idr}
+Forcing I frames as IDR frames.
+
+@item @var{low_power}
+For encoders set this flag to ON to reduce power consumption and GPU usage.
+@end table
+
+@subsection H264 and HEVC Options
```
Comment (xhaihao) [inline doc/encoders.texi]: If a h264 specific option (e.g. `max_frame_size`)  is changed to a common option for h264/hevc in the future, we will have to modify the description for two subsections. In addition, user has to check 3 subsections for all options for h264 and hevc encoders, so  I prefer to add these options in the subsections for h264 and hevc and remove this subsection.

Comment (wenbinc-Bin) [reply doc/encoders.texi]: Ok, I will update it.

**Comment by xhaihao** [comment]
> look_ahead_depth

`look_ahead_depth` is used to enable la in extbrc for hevc encoder, look_ahead is not required.

**Comment by wenbinc-Bin** [comment]
> > look_ahead_depth
> 
> `look_ahead_depth` is used to enable la in extbrc for hevc encoder, look_ahead is not required.

Got it, thank you.

File: doc/encoders.texi (hunk 1 above)
Comment (wenbinc-Bin) [reply doc/encoders.texi]: Done

**Comment by wenbinc-Bin** [comment]
> > > look_ahead_depth
> > 
> > 
> > `look_ahead_depth` is used to enable la in extbrc for hevc encoder, look_ahead is not required.
> 
> Got it, thank you.

I have added look_ahead_depth to hevc doc as well.

**Review by xhaihao — APPROVED** [review]
LGTM

**Comment by wenbinc-Bin** [comment]
submit to upstream: [https://patchwork.ffmpeg.org/project/ffmpeg/patch/20220106030040.781111-1-wenbin.chen@intel.com/](https://patchwork.ffmpeg.org/project/ffmpeg/patch/20220106030040.781111-1-wenbin.chen@intel.com/ )

**Pull request closed without merging**